manifests/repo/redhat.pp

```diff
@@ -7,12 +7,13 @@
 #
 
 class php::repo::redhat (
-  String[1] $yum_repo = 'remi_php56',
+  String[1] $version = '5.6',
```

Comment:
Wondering if we should use an explicit list of allowed values (`Enum['5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1', '8.2']`) which makes it obvious when reading the doc what is allowed here but needs to be updated quite often, or match something like `Pattern['\A\d+\.\d+\z']`?

While speaking about the doc strings, pleas update them in the top of the file.

Reply:
Sure, I can add that

Reply:
I think if anything, the Pattern would make the most sense